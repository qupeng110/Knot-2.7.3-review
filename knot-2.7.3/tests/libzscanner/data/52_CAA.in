$ORIGIN	.
$TTL	1

; OK
@	CAA	0	a	a		; The simplest variant
@	CAA	255	a	a		; Maximal flags
@	CAA	0       a	"a ; b"		; Quoted value
@	CAA	0       a	012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789 ; Value longer than 255
@	TYPE257	\# 4 00016161			; TYPE + Hexadecimal rdata
@	TYPE257	0	a	a		; TYPE
@	caa	0	a	a		; Type in lower-case

; OK fallbacks and extensions
@	CAA	0	""	a		; Empty tag
@	CAA	0	"a"	a		; Quoted tag
@	CAA	0	abcdefghijklmnopqrstuvwxyz0123456789	a	; All allowed characters, longer than 15

; KO
@	CAA
@	CAA					; Empty rdata
@	CAA	\# 0				; Hex empty rdata
@	CAA	256	a	a		; Flags overflow
@	CAA	0	a	a	a	; Unexpected item
