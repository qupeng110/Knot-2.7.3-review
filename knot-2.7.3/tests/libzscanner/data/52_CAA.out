OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=FF016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161203B2062
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=000161303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839303132333435363738393031323334353637383930313233343536373839
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=000061
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00016161
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0101
RDATA=00246162636465666768696A6B6C6D6E6F707172737475767778797A3031323334353637383961
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_NUMBER
------
WARNG=ZS_BAD_HEX_CHAR
------
WARNG=ZS_NUMBER8_OVERFLOW
------
WARNG=ZS_BAD_REST
------
